commands/audit/scarunner.go

```diff
@@ -117,8 +133,6 @@ func getRequestedDescriptors(params *AuditParams) map[techutils.Technology][]str
 	requestedDescriptors := map[techutils.Technology][]string{}
 	if params.PipRequirementsFile() != "" {
 		requestedDescriptors[techutils.Pip] = []string{params.PipRequirementsFile()}
-		// We leverage the ability to pass a custom descriptor name for Conan through PipRequirementsFile as well, therefore we set also Conan with the provided descriptor name.
-		requestedDescriptors[techutils.Conan] = []string{params.PipRequirementsFile()}
```

Comment:
Is this removal on purpose?

Reply:
yes, this is a bug fix for conan, I will add this change to the PR details